test/files/jvm/annotations.check

```diff
@@ -61,3 +62,67 @@ public void Test4$Foo12.name_$eq(java.lang.String)
 99
 dylan
 2
+#partest !java8
```

Comment:
It wouldn't be brain surgery for `partest --update-check` to support existing filters, or possibly under a flag `--check-jvm`.